ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-release-1.20__4.6.yaml

```diff
@@ -0,0 +1,165 @@
+base_images:
+  base:
+    name: "4.6"
+    namespace: ocp
+    tag: base
+  cli:
+    name: "4.6"
+    namespace: ocp
+    tag: cli
+  openshift_knative-v0.26.0_knative-eventing-kafka-src:
+    name: knative-v0.26.0
+    namespace: openshift
+    tag: knative-eventing-kafka-src
+  openshift_knative-v0.26.0_knative-eventing-src:
+    name: knative-v0.26.0
+    namespace: openshift
+    tag: knative-eventing-src
+  openshift_knative-v0.26.0_knative-serving-src:
+    name: knative-v0.26.0
+    namespace: openshift
+    tag: knative-serving-src
+  openshift_release_golang-1.16:
+    name: release
+    namespace: openshift
+    tag: golang-1.16
```

Comment:
All of these should be 1.17 at this point, I think? (all references to Golang 1.16 I meant)

Reply:
Ok I was about to ask about go version because for example for 0.26 Serving and S-O are using 1.16.